code/modules/mob/mob_helpers.dm

```diff
@@ -87,7 +91,9 @@ mob/proc/remove_internal_organ()
 	var/list/difference = list()
 	if(client.color)
 		difference = difflist(client.color,colour_to_apply)
-	if(!difference.len || difference || !istype(difference))
+	if(!difference) // otherwise !difference.len throws a runtime since null.len isn't a thing
+		return
+	if(!difference.len || !istype(difference))
```

Comment:
I assume this is nonsensical celt code aka not your fault but either `!difference` or `!istype(difference)` can/should be removed